cockatrice/src/window_main.cpp

```diff
@@ -60,7 +60,7 @@
 #define GITHUB_PAGES_URL "https://cockatrice.github.io"
 #define GITHUB_CONTRIBUTORS_URL "https://github.com/Cockatrice/Cockatrice/graphs/contributors?type=c"
 #define GITHUB_CONTRIBUTE_URL "https://github.com/Cockatrice/Cockatrice#cockatrice"
-#define GITHUB_TRANSLATOR_RECOGNIZE_URL "https://github.com/Cockatrice/Cockatrice/wiki/Translators"
+#define TRANSIFEX_COLLABORATORS_URL "https://www.transifex.com/cockatrice/collaborators/?"
```

Comment:
This brings me to a "manage contributors" screen if I'm logged in, or brings me to a login page if I'm logged out.

Does it show just a list if you're not a project admin?